zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -123,20 +140,35 @@ private IndexSettings getIndexSettings(final String indexName) {
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndices(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentIndices =
-        searchEngineClient.getMappings(config.getIndex().getPrefix() + "*", MappingSource.INDEX);
+    if (indexDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentIndices = searchEngineClient.getMappings(alLIndexNames(), MappingSource.INDEX);
 
     return schemaValidator.validateIndexMappings(currentIndices, indexDescriptors);
   }
 
   private Map<IndexDescriptor, Collection<IndexMappingProperty>> validateIndexTemplates(
       final IndexSchemaValidator schemaValidator, final SearchEngineClient searchEngineClient) {
-    final var currentTemplates = searchEngineClient.getMappings("*", MappingSource.INDEX_TEMPLATE);
+    if (indexTemplateDescriptors.isEmpty()) {
+      return Map.of();
+    }
+
+    final var currentTemplates =
+        searchEngineClient.getMappings(
+            config.getIndex().getPrefix() + "*", MappingSource.INDEX_TEMPLATE);
 
     return schemaValidator.validateIndexMappings(
         currentTemplates,
         indexTemplateDescriptors.stream()
             .map(IndexDescriptor.class::cast)
             .collect(Collectors.toSet()));
   }
+
+  private String alLIndexNames() {
+    return indexDescriptors.stream()
+        .map(descriptor -> descriptor.getFullQualifiedName() + "*")
```

Comment:
I think the start we don't need

Reply:
we need the wildcard, without it the requests will throw an exception if one of the values doesn't exist, e.g. `getIndices("a,b,c")` if only a and b exist it will throw but if `getIndices(a*,b*c*)` then it will return the a and b indices.

There isn't to my knowledge a better way to check which indices from a list exist in ELS/OS (with regards to doing it in only 1 request). There's [this](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-template-exists-v1.html) which tells you about existence but it only returns a true/false and not more information (which would be ideal)